src/appengine/handlers/testcase_detail/show.py

```diff
@@ -508,7 +508,7 @@ def get_testcase_detail(testcase):
       'crash_address':
           crash_address,
       'crash_state':
-          crash_state,  # Used by reproduce tool.
+          crash_state,
```

Comment:
If reproduce tool was the only use, should we get rid of this. or leave a todo?

Reply:
I think this is useful enough to keep.